canonical-web-and-design/netplan.io
# Replace Jekyll with 11ty

## Done

- Replace Jekyll with 11ty.
- Original `karmdown` library used by Jekyll does not seem to be available with 11ty, replaced by `markdown-it` (and its plugins) to make it work.
- Updated markup to be compatible with 11ty
- Updated ignore file for `dotrun`
- Updated `dockerfile` with new setup using 11ty.

## QA

1. Checkout this branch
2. `dotrun` and browse http://localhost:8024/
3. Check that navigation links work correctly
4. Check that Table of Contents appear correctly
5. Check that global navigation appears.

Fixes #100
